alacritty/src/macos/proc.rs

```diff
@@ -1,41 +1,84 @@
-use std::ffi::{CStr, CString};
+use std::ffi::{CStr, CString, IntoStringError};
+use std::fmt::{self, Display, Formatter};
 use std::io;
 use std::mem::{self, MaybeUninit};
-use std::os::raw;
+use std::os::raw::{c_int, c_void};
 use std::path::PathBuf;
 
-pub fn cwd(pid: raw::c_int) -> io::Result<PathBuf> {
+/// Error occuring during getting cwd from `proc_pidinfo`.
+#[derive(Debug)]
+pub enum Error {
+    /// IO error using `proc_pidinfo`.
```

Comment:
Probably best to just say this is an IO Error. At which point the comment becomes rather useless.